store/Cargo.toml

```diff
@@ -30,7 +30,7 @@ miden-node-utils = { path = "../utils", version = "0.2" }
 miden-objects = { workspace = true }
 once_cell = { version = "1.18.0" }
 prost = { version = "0.12" }
-rusqlite = { version = "0.30", features = ["array", "buildtime_bindgen"] }
+rusqlite = { version = "0.30", features = ["array", "buildtime_bindgen", "bundled"] }
```

Comment:
Should we bump-up the version to v0.31 (I think that's the latest one now).

Reply:
Not for the moment, Deadpool has the SQLite version pinned